mep3_msgs/srv/DynamixelCommand.srv

```diff
@@ -0,0 +1,6 @@
+float64 position   # rad
+float64 velocity   # rad/s
+float64 tolerance  # rad
```

Comment:
Prebaciti u stepene [deg] jer to direktno lako ocitamo iz Dynamixel wizard-a.

Reply:
@lukicdarkoo Ovde si ti glavni, pa deder daj misljenje, u Webots je u "rad", jel tako, pa si zato stavio? Sto se mene tice svejedno je ko konvertuje, ti odluci...

Ali s obzirom na to da cemo mozda uz pomoc Dynamixel Wizard-a nesto kalibrisati, proveravati uglove, mozda je Dusanova ideja bolja? Takodje za razmisljanje i pisanje zadataka mislim da su nam prirodniji stepeni?

Reply:
> Ovde si ti glavni

Nema glavnog kada su tehnicke odluke u pitanju :) U Webots-u jeste rad, ali to se lako konvertuje. Stepeni su malo intuitivniji za testiranje i pisanje strategija.

Reply:
Koristi `math.radians(x)` i `math.degrees(x)` za konverzije